console/console_ftp.py

```diff
@@ -66,12 +76,20 @@ def get_folder_contents(self, path):
                 if fname not in ('.', '..', '.DS_Store') and bits[2].isdigit():
                     meta = {
                         'name': fname,
-                        'modify': datetime.strptime(date_string, '%m-%d-%y %I:%M%p').isoformat(),
-                        'size': int(bits[2])
                     }
+
+                    try:
+                        meta['modify'] = datetime.strptime(date_string, '%m-%d-%y %I:%M%p').isoformat()
+                        meta['size'] = int(bits[2])
+                    except Exception:
```

Comment:
Lets not catch top level exception and instead catch the specific errors thrown in this scenario